src/Swashbuckle.AspNetCore.SwaggerGen/Annotations/XmlCommentsOperationFilter.cs

```diff
@@ -98,7 +99,13 @@ private void ApplyResponsesXmlToResponses(IDictionary<string, Response> response
             {
                 // Check for a corresponding  API parameter (from ApiExplorer) that's property-bound?
                 var propertyParam = apiDescription.ParameterDescriptions
-                    .Where(p => p.ModelMetadata?.ContainerType != null && p.ModelMetadata?.PropertyName != null)
```

Comment:
@Bjego, here double check is done that can be eliminated.
If you check for null than do something once `?.` is nice but if you apply `?.` to the same property more than once it's a waste of resources.
This short form does not add any value in readability too.